src/app/components/Modals/ClaimSuccessModal.tsx

```diff
@@ -0,0 +1,65 @@
+import { Text } from "@babylonlabs-io/core-ui";
+
+import { getNetworkConfigBBN } from "@/config/network/bbn";
+import { getNetworkConfigBTC } from "@/config/network/btc";
+
+import { SubmitModal } from "./SubmitModal";
+
+interface ClaimSuccessModalProps {
+  open: boolean;
```

Comment:
```suggestion
 loading: boolean;
  open: boolean;
```

Too many modals for a such small feature. Can we combine 2 into 1?